R/translate.R

```diff
@@ -162,20 +162,24 @@ translate_part <- function(xml, glossary_id, source_lang, target_lang, formality
   ## protect content inside curly braces ----
   woolish$body <- tinkr::protect_curly(woolish$body)
   curlies <- xml2::xml_find_all(woolish$body, "//*[@curly]")
-  replace_curly <- function(curly) {
-    xml2::xml_name(curly) <- "curly"
+  purrr::walk(curlies, protect_curly)
+
+  ## protect content inside square brackets ----
+  contain_square_brackets <- xml2::xml_find_all(
+    woolish$body,
+    '//*[contains(text(), "[") and contains(text(), "[")]'
```

Comment:
@yabellini so if one inputs text where the square brackets are on different lines, the protection won't work.